packages/hash/api/src/model/entityType.model.ts

```diff
@@ -234,13 +234,25 @@ class __EntityType {
     return dbEntityTypes.map((entityType) => new EntityType(entityType));
   }
 
-  static async getEntityTypeParents(
-    client: DBClient,
-    params: { entityTypeId: string },
-  ) {
-    const dbEntityTypes = await client.getEntityTypeParents(params);
+  async getParentEntityTypes(client: DBClient) {
```

Comment:
Perhaps it's worth enforcing the return type
```suggestion
  async getParentEntityTypes(client: DBClient): Promise<EntityType[]> {
```